src/Cardano/Wallet/Primitive.hs

```diff
@@ -261,3 +291,12 @@ newtype Hash (tag :: Symbol) = Hash
     } deriving (Show, Generic, Eq, Ord)
 
 instance NFData (Hash tag)
+
+
+-- | A polymorphic wrapper type with a custom show instance to display data
+-- through 'Buildable' instances.
+newtype ShowFmt a = ShowFmt a
```

Comment:
this one is because we do not want to deliver separate instances of Show for Utxo, Adresses, Block,... Instead we wrap them in ShowFmt and in one go use any type's Buildable instance?

Reply:
This one is because we do not want to write custom show instances in practice. Though, because of the way QuickCheck, HSpec and some other libraries work, they use `Show` to output results and counterexamples which is very unpractical to read for some types (like the UTxO). For those, we may opt-in for a `ShowFmt` wrapper around a type to get a nice output. Though, we don't want to be changing the stock `Show` instance for those types.